terraform/locals.tf

```
@@ -1,5 +1,5 @@
 locals {
   resource_prefix = "pocketddd-${var.env}"
   sql_server_name = "${local.resource_prefix}-sql-server"
-  subdomain = var.env == "production" ? "pocket" : "pocket-${var.env}"
+  subdomain = var.env == "pocket2024" ? "pocket" : "pocket-${var.env}"
```

shouldn't this be something like:
```suggestion
  subdomain = var.env == "production" ? "pocket2024" : "pocket-${var.env}"
```
or have I lost the plot 😆